app/components/publications/publication-sidebar.hbs

```diff
@@ -36,31 +37,65 @@
               </div>
             {{/if}}
           </div>
+
+          {{!-- Status --}}
           <Publications::PublicationStatusSelector
                   @publicationStatus={{this.publicationStatus}}
                   @onChange={{this.setPublicationStatus}}
+                  @disabled={{or this.publicationStatus.isPublished this.publicationStatus.isWithdrawn}}
```

Comment:
Ik moet toegeven dat de selectors in het prototype er disabled uit zien, maar ze zijn het niet (test maar ...). In een ideaal scenario zouden deze disable's ook perfect logisch zijn, maar gezien de gebruikers regelmatig onvoorziene situaties behandelen, willen we de flexibiliteit behouden om deze elementen ten allen tijde aanpasbaar te houden. Ooit doen we hier misschie nog wat fine-tuning aan, maar ik zou de `@disabled`'s voor nu weglaten.